classes/main.rb

```diff
@@ -0,0 +1,87 @@
+require_relative 'app'
+require_relative 'execute'
+require 'json'
+
+class Main
+  def display_ui
+    puts 'Please choose an option by entering a number:'
+    load_data
+    puts '1 - List all books'
+    puts '2 - List all people'
+    puts '3 - Create a person'
+    puts '4 - Create a book'
+    puts '5 - Create a rental'
+    puts '6 - List all rentals for a given person id'
+    puts '7 - Save data And Exit'
+    user_choice = gets.chomp.to_i
+    puts user_choice
+    Execute.new(user_choice).execute
+  end
+
+  # rubocop:disable Metrics/MethodLength
+  # rubocop:disable Metrics/AbcSize
+  # rubocop:disable Metrics/CyclomaticComplexity, Metrics/PerceivedComplexity
```

Comment:
- [ ] Hi, can you enable Rubocop? because, [Rubocop](https://www.rubocop.org/) is a Ruby static code analyzer (a.k.a. linter) and code formatted,  a tool that analyzes your source code to flag programming errors, bugs, stylistic errors, and suspicious constructs.
- [ ] I would advise you to make sure the form of UI is created and working after you finish solving the linter baggs.